### PR TITLE
Update index.yaml

### PR DESCRIPTION
**Pull request by kingamarton**

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -39,10 +39,10 @@ entries:
         - https://github.com/apache/incubator-yunikorn-release/releases/download/v0.8.0/yunikorn-0.8.0.tgz
       version: 0.8.0
     - apiVersion: v1
-      appVersion: 0.9.0
-      created: "2020-06-24T14:12:46.578917+02:00"
+      appVersion: latest
+      created: "2020-07-17T12:19:19.969474+02:00""
```
Comment (yangwwei) [inline index.yaml:43]: there is an extra quote, pls remove it.

File: index.yaml [hunk 2]
```diff
@@ -39,10 +39,10 @@ entries:
         - https://github.com/apache/incubator-yunikorn-release/releases/download/v0.8.0/yunikorn-0.8.0.tgz
       version: 0.8.0
     - apiVersion: v1
-      appVersion: 0.9.0
-      created: "2020-06-24T14:12:46.578917+02:00"
+      appVersion: latest
```
Comment (yangwwei) [inline index.yaml:42]: is it OK to set appVersion to latest instead of a numeric version?

Comment (kingamarton) [reply index.yaml:42]: The app version is an optional field and it is the version of the application the chart contains, in our case the YK version

File: index.yaml (hunk 1 above)
Comment (kingamarton) [reply index.yaml]: done

**Pull request merged**